### PR TITLE
Fix Debian, Ubuntu multiarch building

### PR DESCRIPTION
**Pull request by muzena**

### CODE REVIEW

File: Packaging/fedora/devilutionx.desktop [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Name=devilutionX
 GenericName=DevilutionX
 Comment=Play Diablo I on Linux
+Comment[hr]=Igrajte Diablo I na Linuxu
```
Comment (AJenbo) [inline Packaging/fedora/devilutionx.desktop]:
```suggestion
Comment[da]=Spil Diablo I på Linux
Comment[hr]=Igrajte Diablo I na Linuxu
```

**Comment by AJenbo** [comment]
Does this build with `-DNIGHTLY_BUILD=ON`? Without it you will have a lot of extra dependencies.

**Comment by muzena** [comment]
I will look tommorow in build log.

**Comment by muzena** [comment]
Can you now look [log](https://launchpadlibrarian.net/459126345/buildlog_ubuntu-bionic-amd64.devilutionx_1.0.0+git908-534ed2d-202001041832~ubuntu18.04.1_BUILDING.txt.gz), I added in [rules](https://github.com/muzena/devilutionX/blob/debian-packaging_new/Packaging/debian/rules) file -DNIGHTLY_BUILD=ON option.

**Comment by AJenbo** [comment]
> Can you now look [log](https://launchpadlibrarian.net/459126345/buildlog_ubuntu-bionic-amd64.devilutionx_1.0.0+git908-534ed2d-202001041832~ubuntu18.04.1_BUILDING.txt.gz), I added in [rules](https://github.com/muzena/devilutionX/blob/debian-packaging_new/Packaging/debian/rules) file -DNIGHTLY_BUILD=ON option.

Looks good

**Pull request merged**